postgres/postgres_pods_dashboard.json

```diff
@@ -1613,18 +1613,18 @@
           "value": "ps-demo"
         },
         "datasource": "${datasource}",
-        "definition": "query_result(kubedb_postgres_created{namespace=\"$namespace\"})",
+        "definition": "query_result(kubedb_com_postgres_created{namespace=\"$namespace\"})",
         "description": null,
         "error": null,
         "hide": 0,
         "includeAll": false,
         "label": "postgres",
         "multi": false,
-        "name": "db",
```

Comment:
https://github.com/appscode/grafana-dashboards/blob/c10aeebabce90a0381864f2b4afa969a7c77227c/postgres/postgres_pods_dashboard.json#L1644

https://github.com/appscode/grafana-dashboards/blob/c10aeebabce90a0381864f2b4afa969a7c77227c/postgres/postgres_pods_dashboard.json#L1653

These two `${db}` variables should be replaced by `${app}`

Reply:
Fixed here https://github.com/appscode/grafana-dashboards/pull/21/commits/f208ec5283e1230676223fbe5ec23e7c329cc37c